### DISCUSSION TITLE
Where is the User Guide

### DISCUSSION THREAD

**Discussion by fitsou** [matt8707/ha-fusion#414] [Q&A]
Hello to all, I've installed the HA-Fusion with the intention to use it as a Kyosk in an old iPad.
The installation path went smoothly, but then nothing.. I see no user guide, no documentation, nothing.

How can I add Home Assistance entities,etc into HA-Fusion?

**Comment by anarion80** [comment]
There is no user guide really. If you have it installed and running, just click on the down arrow in the upper-right corner, then Edit UI:
![image](https://github.com/matt8707/ha-fusion/assets/2185791/24c015a6-7eef-425b-abdb-0f5b5e4d385a)
then Add:
![image](https://github.com/matt8707/ha-fusion/assets/2185791/9b1f99b6-17fe-4d6c-8fec-cead16101244)
and start playing with it.

**Comment by fitsou** [comment]
The only available option is the add view. I cannot find a way to add entities example the lights from HA

**Reply by TheOpsMgr** [reply]
Been playing with it today and AFAIK the sequence is 

1. Add a View
2. Add a Section
3. Add Object eg Button. Object option is only available when a section is on a View.
4. Edit the Object properties to select the Entity you want to control.
5. Note - if it's something like a Switch and you want it to toggle immediately it's pressed, set "Show More Information" to NO. Otherwise, it will open up a new dialog screen, with a Toggle radio button, which just adds an unnecessary step.

![image](https://github.com/matt8707/ha-fusion/assets/3353242/1fe17e38-fcdb-4f03-92ba-f4a5463d86ba)

![image](https://github.com/matt8707/ha-fusion/assets/3353242/3514a717-64a2-4bcf-a771-0e9f475bbeab)

**Comment by fitsou** [comment]
How do you rename a Section?
And how can I make the row 2x2 in each Section?

**Comment by TheOpsMgr** [comment]
Just click on the default text "Section" and type away!

![image](https://github.com/matt8707/ha-fusion/assets/3353242/54e14b17-ff91-43bb-98bb-6caf83cf0815)

**Comment by TheOpsMgr** [comment]
I don't know how you create a fixed 2x2 grid. My assumption is that because the UI is supposed to be responsive ie dynamically changing to device / screen size / orientation that "fixed" elements are not available.

That said, you can add more and more Sections into a Horizontal Stack to change the number of columns but, again, how this looks on a different device is anyone's guess. There is also at "Empty" object you can use to create spaces and gaps to make things look nicer.

![image](https://github.com/matt8707/ha-fusion/assets/3353242/d915cf75-958d-4bd6-a52b-c390550fefa6)
